notebooks/visualization/Cartopy-Intro.ipynb

```diff
@@ -0,0 +1,389 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>Introduction to Cartopy</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "<div style=\"float:right; width:1200 px\"><img src=\"../../instructors/images/cartopy_example.png\" alt=\"sin(x) with few data points\" style=\"height: 300px;\"></div>\n",
```

Comment:
This is not referencing the correct file name, it should be pointed at "cartopy-basic-preview.png", also because of that, it made it easy to notice that the "alt" text should be updated as well since the preview is not of a sin(x) graph.